File: iniciante/javascript/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var entrada1 = lines.shift().split(' ')
var entrada2 = lines.shift().split(' ')

var valorAPagar = parseInt(entrada1[1]) * parseFloat(entrada1[2]) + parseInt(entrada2[1]) * parseFloat(entrada2[2])

console.log(`VALOR A PAGAR: R$ ${valorAPagar.toFixed(2)}`)

File: iniciante/python/1010.py
# -*- coding: utf-8 -*-

entrada1 = input()
entrada2 = input()

lista_entrada1 = entrada1.split(' ')
lista_entrada2 = entrada2.split(' ')

valor_pagar = int(lista_entrada1[1]) * float(lista_entrada1[2]) + \
              int(lista_entrada2[1]) * float(lista_entrada2[2])

print(f'VALOR A PAGAR: R$ {valor_pagar:.2f}')
